src/lib_rules.ml

```diff
@@ -287,15 +287,15 @@ module Gen (P : Install_rules.Params) = struct
         ]
     in
     List.map lib.c_names ~f:(fun name ->
-      build_c_file   lib ~expander ~dir ~includes (resolve_name name ~ext:".c")
+      build_c_file   lib ~dir ~includes (resolve_name name ~ext:".c")
     ) @ List.map lib.cxx_names ~f:(fun name ->
-      build_cxx_file lib ~expander ~dir ~includes (resolve_name name ~ext:".cpp")
+      build_cxx_file lib ~dir ~includes (resolve_name name ~ext:".cpp")
```

Comment:
Passing expanders is preferable. It's a bit tedious at the moment but we're trying to minimize uses of Super_context.

Reply:
fixed.